changelog

```diff
@@ -1,3 +1,202 @@
+      * Removed duplicate Korath phrase. (@tibetiroka)
+      * Clarified the description of the Far Osk 27. (@ziproot)
+      * Removed outfit scanning capabilities from Grey's person ship. (@mOctave)
+      * Blocked "Lunarium: Questions" from completing while assisting the Heliarchs. (@ziproot)
+      * Reworded Deep's description. (@roadrunner56)
+      * Listed correct attributions for Unsplash images. (@ziproot)
+      * Added back instances of "ser" and explained what it means in the Skadenga and Stones of Our Fathers missions. (@bene-dictator)
+      * The Terraforming missions are no longer offered at non-sensical times. (@lumbar527)
+      * Fixed an inconsistency in condition names used by Patir Mystery and Cognizance. (@OcelotWalrus)
+      * Don't select Graffias as a target for Free Worlds "Scouting Run" jobs. (@warp-core)
+      * "Deep: Remnant: Continue Research" no longer offers if you've completed the missions after it. (@Saugia)
+      * Adjusted the explosion stats for the Korsmanath A'awoj and Echo-galleon. (@Quantumshark)
+      * Updated an outdated news item about human drones. (@tibetiroka)
+      * Pirate jobs with illegal cargo are now marked as "infiltrating". (@ziproot)
+    * Engine bugs:
+      * Fix edge case in code style checker. (@tibetiroka)
+      * Don't consider "Parsing: <file>" output when in debug mode as errors when test parsing. (@warp-core)
+      * Fix game crash on short error messages. (@tibetiroka)
+      * Don't draw name tooltips for ships hidden under the shop sidebar footer. (@TomGoodIdea)
+      * Variant ships correctly inherit display names from base ships. (@TomGoodIdea)
+      * Use singular "ship" in messages when recalling/deploying a single ship. (@tibetiroka)
+      * Fix errors in the copyright file. (@tibetiroka)
+      * Rename variable to avoid ambiguity with std::queue in some versions of clang. (@quyykk)
+      * Fix ship parenting on takeoff with fighter flagship. (@tibetiroka)
+      * Correctly handle arrow keys with no item selected in the shop. (@TomGoodIdea)
+      * Align plugin checkbox clickzones correctly. (@warp-core)
+      * Fix mission distance calculations with different settings. (@warp-core)
+      * Fix crashes when loading/unloading images. (@quyykk)
+      * Fix custom port button name when using "add port". (@warp-core)
+      * Corrected the parsing of the initial player reputation grandchild node in Government. (@xxxyyyqqq12345)
+      * Draw distance to the selected system after mission markers. (@TomGoodIdea)
+      * Ships with "disables" personality ask for help disabling, instead of destroying, their enemies. (@TomGoodIdea)
+      * Suppress "missions require landing in the next system" messages from failed missions. (@OcelotWalrus)
+      * Properly check if a mission is failed. (@TomGoodIdea)
+      * Hardpoint labels and lines in the ship info panel will properly align to the "weapons" box from the interface definition. (@Zitchas)
```

Comment:
```suggestion
      * Fixed edge case in code style checker. (@tibetiroka)
      * Don't consider "Parsing: <file>" output when in debug mode as errors when test parsing. (@warp-core)
      * Fixed game crash on short error messages. (@tibetiroka)
      * Don't draw name tooltips for ships hidden under the shop sidebar footer. (@TomGoodIdea)
      * Variant ships now correctly inherit display names from base ships. (@TomGoodIdea)
      * Use singular "ship" in messages when recalling/deploying a single ship. (@tibetiroka)
      * Fixed errors in the copyright file. (@tibetiroka)
      * Renamed variable to avoid ambiguity with std::queue in some versions of clang. (@quyykk)
      * Fixed ship parenting on takeoff with fighter flagship. (@tibetiroka)
      * Correctly handle arrow keys with no item selected in the shop. (@TomGoodIdea)
      * Aligned plugin checkbox clickzones correctly. (@warp-core)
      * Fixed mission distance calculations with different settings. (@warp-core)
      * Fixed crashes when loading/unloading images. (@quyykk)
      * Fixed custom port button name when using "add port". (@warp-core)
      * Corrected the parsing of the initial player reputation grandchild node in Government. (@xxxyyyqqq12345)
      * Draw distance to the selected system after mission markers. (@TomGoodIdea)
      * Ships with "disables" personality now ask for help disabling, instead of destroying, their enemies. (@TomGoodIdea)
      * Suppress "missions require landing in the next system" messages from failed missions. (@OcelotWalrus)
      * Properly check if a mission is failed. (@TomGoodIdea)
      * Hardpoint labels and lines in the ship info panel now properly align to the "weapons" box from the interface definition. (@Zitchas)
```